docs/how-to/integrations/machine-translation.md

```diff
@@ -124,6 +124,8 @@ WAGTAILLOCALIZE_MACHINE_TRANSLATOR = {
     "CLASS": "wagtail_localize.machine_translators.deepl.DeepLTranslator",
     "OPTIONS": {
         "AUTH_KEY": "<Your DeepL key here>",
+        # Optional DeepL API option
+        "FORMALITY": "<Your DeepL formality preference here>",
```

Comment:
Reading https://www.deepl.com/docs-api/translate-text we should
1. link to the docs
2. highlight that only "default", "prefer_less", "prefer_more" will work across all locales.
Even better, we should only accept these three

Reply:
Sounds good!

How would you like me to handle the case in which someone provides an unsupported formality option?

- Raise a ValueError
- Ignore the provided option and don't set formality

I personally prefer the first

Reply:
I'd say raise a warning and ignore

Reply:
Updated 👍